peril-settings.json

```diff
@@ -8,32 +8,34 @@
     "rules": {},
     "repos": {
         "wordpress-mobile/WordPress-iOS": {
-            "pull_request, pull_request.labeled, pull_request.unlabeled": [
+            "pull_request": [
```

Comment:
I'd love to reduce that filter even more, to only run the platform-specific rule (`ios-macos.ts` or `android.ts` depending on the repos) and the milestone rule only on events that makes sense. But:

## For the platform-specific rule

Since the rule is pretty generic (i.e. it contains many different checks), it feels safer to run it on every action. One would have to check the rule implementation in details to check what GitHub/Peril APIs we use on those rules (i.e. if we access the list of labels, the diff, etc) to know what filter would be appropriate, but given that rule is quite generic by nature and its implementation might evolve with new checks while not changing the rule name, it felt quite risky to tight it to only specific actions.

This could be discussed further, and we could at least consider only limiting to `pull_request.opened, pull_request.synchronize`, which would probably make sense (I'd be ok to change it to that, I'm just not sure we wouldn't miss any other action we wouldn't have thought of there), and would have the benefit to avoid `pull_request.closed` (which we definitively don't need there).

What do you think? If we limit to specific actions, what should be the full list for this rule so we don't forget any case?

## Milestone rule

I haven't found any action name for when a milestone is added. In fact, I tried to change the milestone of an opened PR, then went to look at [the delivered webhook payloads in GitHub](https://github.com/wordpress-mobile/WordPress-Android/settings/hooks/85494651/deliveries) and didn't see any event triggered from that milestone addition/removal. And the [list of actions for the `pull_request` event in GitHub API docs](https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types#pullrequestevent) don't mention any action that could match the milestone addition either. This is why I ended up leaving this one under the generic `pull_request` without any specific action, as I don't know which one we should use if any.

In practice I think what's happening (even currently) is that adding/removing a milestone don't make Peril re-check for this rule (like we'd hope it does), and we need to do something else to the PR (like pushing new commits or adding labels) to force-retrigger it. So in this sense, this PR doesn't change that existing behavior.

Like for the platform rule, I'm opened to suggestions as to how we can improve that and which actions could be reasonable to filter on for this rule.